entity-framework/core/miscellaneous/1x-2x-upgrade.md

```diff
@@ -10,29 +10,25 @@ uid: core/miscellaneous/1x-2x-upgrade
 
 We have taken the opportunity to significantly refine our existing APIs and behaviors in 2.0. There are a few improvements that can require modifying existing application code, although we believe that for the majority of applications the impact will be low, in most cases requiring just recompilation and minimal guided changes to replace obsolete APIs.
 
-## Procedures Common to All Applications
-
 Updating an existing application to EF Core 2.0 may require:
 
-1. Upgrading the target .NET platform of the application to one that supports .NET Standard 2.0. See [Supported Platforms](../platforms/index.md) for more details.
+1. Upgrading the target .NET implementation of the application to one that supports .NET Standard 2.0. See [Supported .NET Implementations](../platforms/index.md) for more details.
 
 2. Identify a provider for the target database which is compatible with EF Core 2.0. See [EF Core 2.0 requires a 2.0 database provider](#ef-core-20-requires-a-20-database-provider) below.
 
 3. Upgrading all the EF Core packages (runtime and tooling) to 2.0. Refer to [Installing EF Core](../get-started/install/index.md) for more details.
 
-4. Make any necessary code changes to compensate for breaking changes. See the [Breaking Changes](#breaking-changes) section below for more details.
+4. Make any necessary code changes to compensate for the breaking changes described in the rest of this document.
+
+## ASP.NET Core now includes EF Core
 
-## ASP.NET Core applications
+Applications targeting ASP.NET Core 2.0 can use EF Core 2.0 without additional dependencies besides third party database providers. However, applications targeting previous versions of ASP.NET Core need to upgrade to ASP.NET Core 2.0 in order to use EF Core 2.0. For more details on upgrading ASP.NET Core applications to 2.0 see [the ASP.NET Core documentation on the subject](https://docs.microsoft.com/aspnet/core/migration/1x-to-2x/).
 
-1. See in particular the [new pattern for initializing the application's service provider](#new-way-of-getting-application-services) described below.
+## New way of getting application services in ASP.NET Core
 
 > [!TIP]  
```

Comment:
@tdykstra I will make those changes. We already link to IDesignTimeDbContextFactory document. Maybe something didn't work for you?

Reply:
I missed the link because it doesn't work on the [GitHub rendering of the page](https://github.com/aspnet/EntityFramework.Docs/blob/e1f30e322a5a0f9328e768d64a6c69327d629e26/entity-framework/core/miscellaneous/1x-2x-upgrade.md).  We have a [docs guideline not to add formatting to hyperlink text](https://github.com/aspnet/EntityFramework.Docs/blob/e1f30e322a5a0f9328e768d64a6c69327d629e26/entity-framework/core/miscellaneous/1x-2x-upgrade.md), I'm guessing in this case the italics on part of the link text makes it not work in GitHub.  It works on the site though.